docs/analysis-workflows/data-types.md

```diff
@@ -0,0 +1,176 @@
+---
+id: data-types
+title: Data Types Overview
+sidebar_label: Data Types and File Formats
+---
+
+ICGC ARGO provides access to harmonized data from new programs
+
+## Reads
+
+### Submitted Reads
+
+Raw reads submitted by ARGO member programs that are used as inputs to the ARGO analytic pipeline(s). Data is accepted in FASTQ or BAM format and must be compressed.
+
+#### File Types
+
+| Filename Pattern        | Description                             | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ----------------------- | --------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.fq.gz, \*.fastq.gz   | FASTQ file format compressed with gzip  | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.fq.bz2, \*.fastq.bz2 | FASTQ file format compressed with bzip2 | sequencing_reads | Sequencing Reads | N/A                    |
+| \*.bam                  | Binary Alignment Map (BAM) file format  | sequencing_reads | Sequencing Reads | N/A                    |
+
+### Aligned Reads
+
+A read is a sequence obtained from a single sequencing experiment. An aligned read, is a sequence that has been aligned to a common reference genome. Typically these reads can number from the hundreds of thousands to tens of millions. Aligned reads are generated internally by the ARGO alignment workflow(s) and maintained in CRAM (a compressed version of BAM that stores only reads different from the reference sequence) format.
+
+#### File Types
+
+| Filename Pattern | Description                               | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram      | CRAM file format and compressed with gzip | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+### Aligned Reads Index
+
+Secondary files that are external index files for CRAM format files. CRAI files follow a naming convention to correspond to their matching CRAM file of the orginal filename suffixed with `.cram.crai`. Index files are required for selective access to genomic data inside a CRAM via CRAM slicing.
+
+#### File Types
+
+| Filename Pattern | Description                                                 | Analysis Type    | Data Category    | Generating Workflow(s) |
+| ---------------- | ----------------------------------------------------------- | ---------------- | ---------------- | ---------------------- |
+| \*.aln.cram.crai | Cram Index file format. Requires a corresponding CRAM file. | sequencing_reads | Sequencing Reads | DNA Seq Alignment      |
+
+## Quality Control Metrics
+
+Quality control metrics are collected and recorded at several checkpoints in the ARGO Analytics pipeline to ensure that only high-quality data is released. All QC Metrics are associated to a corresponding analysis/file set that they annotate.
```

Comment:
should ARGO Analytics pipeline be `ARGO Analysis pipeline` - that's what the menu item is for the docs site